templates/base.html.twig

```diff
@@ -10,11 +10,17 @@
 	</title>
 	{% block stylesheets %}
 		<link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" />
+		<link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" />
+		
 	{% endblock %}
 	{% block scripts %}
 		<script type="text/javascript" src="//tools-static.wmflabs.org/cdnjs/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
 		<script type="text/javascript" src="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/js/bootstrap.min.js"></script>
+		<script src="sortable.min.js"></script>
```

Comment:
This file doesn't exist. Did you forget to commit it? Or should this be referencing a file in cdnjs?